src/assets/source/scripts/modules/serlo_side_navigation.js

```diff
@@ -287,6 +289,10 @@ define("side_navigation", ["jquery", "underscore", "referrer_history", "events",
                 $ul.append(menuItem.render().$el);
             });
 
+            if (self.levels[0][0].data.parent.data.community) {
```

Comment:
Looks hacky, because the path is static. Are you sure that this works in every scenario, even when the navigation is deeper than 2 levels?